### PR TITLE
Issue 109/Login styling updates (outdated)

### PR DESCRIPTION
**Pull request by andycwilliams**
Starting new branch off of Development. Adding styling only to Login page. Had no issues logging in afterwards.

### CODE REVIEW

File: scratch.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
+/* ----- NEW CLIENT DIALOG BOX -----
```
Comment (leekahung) [inline scratch.js]: I believe this file is not being used currently? I think can remove this file.

File: src/components/Login/Login.jsx [hunk 2]
```diff
@@ -17,24 +22,45 @@ const Login = () => {
 
   return (
     <>
+      <CssBaseline />
```
Comment (leekahung) [inline src/components/Login/Login.jsx]: I believe CssBaseLine is something that should be used in the global index.jsx for MUI's CSS Reset, not here in individual components.

File: src/theme.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import { createTheme } from '@mui/material/styles';
+
+const theme = createTheme({
+  typography: {
+    fontFamily: [
```
Comment (leekahung) [inline src/theme.js]: I am curious as to why there's so many font families in here? Can we not just select one, like "Roboto"

**Review by leekahung — CHANGES_REQUESTED** [review]
Seems to be running fine, login seems to be working as intended.

See feedback for requested changes.

File: scratch.js (hunk 1 above)
Comment (andycwilliams) [reply scratch.js]: Yes it can

File: src/theme.js (hunk 3 above)
Comment (andycwilliams) [reply src/theme.js]: They're used as backups in case the prior ones cannot be loaded, but yeah we can cut down to one or two

**Comment by leekahung** [comment]
Oh, one more thing, please refrain from naming branches with "#". The "#" I gave as an example is meant to be a placeholder for the actual issue number. Like issue #108. So we should really be naming this branch "issue-108/login-styling-updates".

**Pull request closed without merging**